waypoints/Alps-mountains.cup

```diff
@@ -0,0 +1,3886 @@
+"Tscheischhorn",Tscheorn,CH,4625.782N,00931.080E,3019.0m,7,,,,"Gipfel"
+"Tschekelnock",Tscheock,AT,4639.141N,01332.220E,1892.0m,7,,,,"Gipfel"
+"Tschengles",Tscheles,IT,4637.733N,01152.133E,1962.0m,7,,,,"Gipfel"
+"Tschenglser-Hochwand",Tscheand,IT,4633.816N,01037.833E,3375.0m,7,,,,"Gipfel"
+"Tschiernock",Tschiock,AT,4651.850N,01334.083E,2088.0m,7,,,,"Gipfel"
+"Tschigat",Tschigat,IT,4643.133N,01103.516E,3000.0m,7,,,,"Gipfel"
+"Tschimas da Tisch",Tschisch,CH,4636.933N,00949.016E,2872.0m,7,,,,"Gipfel"
+"Tschirgant",Tschiant,AT,4714.350N,01048.000E,2372.0m,7,,,,"Gipfel"
+"Tschoren",Tschoren,IT,4654.200N,01140.150E,2528.0m,7,,,,"Gipfel"
+"Tschuggen",Tschugen,CH,4636.042N,00756.854E,2520.1m,7,,,,"Gipfel"
+"Tulfein",Tulfein,AT,4713.116N,01132.316E,2308.0m,7,,,,"Gipfel"
+"T¸rchlwand",T¸rchand,AT,4709.066N,01301.916E,2577.0m,7,,,,"Gipfel"
+"Turge de la Suffie",Turgefie,FR,4450.150N,00647.050E,2825.0m,7,,,,"Gipfel"
+"T¸rken-Kogel",T¸rkegel,AT,4736.486N,01354.124E,1756.0m,7,,,,"Gipfel"
+"Turntalerkogel",Turntgel,AT,4737.866N,01522.050E,1610.0m,7,,,,"Gipfel"
+"Turracher Hˆhe",Turraˆhe,AT,4654.999N,01352.500E,1783.0m,1,,,,"Wirtshaus s¸dlich vom See"
+"Tuverspitze",Tuvertze,IT,4631.850N,01049.750E,3097.0m,7,,,,"Gipfel"
+"Tuxer Joch",Tuxeroch,AT,4706.105N,01138.060E,2649.9m,7,,,,"Gipfel"
+"‹berseil-Spitze",‹berstze,IT,4655.083N,01132.700E,2493.0m,7,,,,"Gipfel"
+"Uia di Ciamorella",Uiadilla,IT,4519.616N,00708.666E,3676.0m,7,,,,"Gipfel"
+"Uia di Mondrone",Uiadione,IT,4519.399N,00713.083E,2961.0m,7,,,,"Gipfel"
+"Ulrichs-Horn",Ulricorn,AT,4732.683N,01235.916E,2155.0m,7,,,,"Gipfel"
+"Unlaﬂ-Kogel",Unlaﬂgel,AT,4708.166N,01213.616E,3074.0m,7,,,,"Gipfel"
+"Unt. Gabelhorn",Unt.Gorn,CH,4601.500N,00742.016E,3392.0m,7,,,,"Gipfel"
+"Unterberg",Untererg,IT,4658.133N,01141.433E,2990.0m,7,,,,"Gipfel"
+"Unterberg-Horn",Unterorn,AT,4737.150N,01226.266E,1773.0m,7,,,,"Gipfel"
+"Unterer Pfandlscharte",Unterrte,AT,4705.516N,01247.066E,2663.0m,7,,,,"Gipfel"
+"Unterrothorn",Unterorn,CH,4601.166N,00748.000E,3103.0m,7,,,,"Gipfel"
+"Upsspitze",Upssptze,AT,4725.866N,01052.616E,2334.0m,7,,,,"Gipfel"
+"Urbeleskar-Spitze",Urbeltze,AT,4720.233N,01027.916E,2636.0m,7,,,,"Gipfel"
+"Vacheres",Vacheres,FR,4440.516N,00548.066E,2400.0m,7,,,,"Gipfel"
+"Vajolet-T¸rme",Vajolrme,IT,4627.900N,01137.700E,2980.0m,7,,,,"Gipfel"
+"Valchevriere",Valchere,FR,4504.089N,00528.633E,1583.0m,7,,,,"Gipfel"
+"Vallaca",Vallaca,IT,4622.716N,01136.450E,2319.0m,7,,,,"Gipfel"
+"Valluga",Valluga,AT,4709.466N,01012.833E,2809.0m,7,,,,"Sb-Gipfelsta"
+"Valvelspitze",Valvetze,IT,4646.183N,01040.400E,3356.0m,7,,,,"Gipfel"
+"Vaumuse",Vaumuse,FR,4410.219N,00605.364E,1436.0m,7,,,,"Gipfel"
+"Vd. Langtal-Spitze",Vd.Latze,AT,4700.566N,01247.433E,2310.0m,7,,,,"Gipfel"
+"Vd. Rendel-Spitze",Vd.Retze,AT,4705.950N,01017.500E,2816.0m,7,,,,"Gipfel"
+"Vedretta di Dosegu",Vedreegu,IT,4621.650N,01032.800E,3222.0m,7,,,,"Gipfel"
+"Veitsberg",Veitserg,AT,4734.433N,01158.417E,1787.0m,7,,,,"Gipfel"
+"Veliki Ro
```

Comment:
encoding issue?

Reply:
I tried fixing all the Umlaute. Can you pls have a second look?